gpu/octree/src/cuda/octree_host.cu

```diff
@@ -39,9 +39,11 @@
 
 #include "internal.hpp"
 #include "utils/boxutils.hpp"
+#include "approx_nsearch.hpp"
```

Comment:
`#include "..."` is not considered a good practice. Can we queue this up for a later cleanup PR where we pass proper include directories flags to the compiler in order to enable the use of `#include <pcl/...>`.